cfripper/__version__.py

```diff
@@ -1,3 +1,3 @@
-VERSION = (1, 5, 3)
+VERSION = (1, 5, 4)
```

Comment:
Given that this introduces new rules, we should probably increase a full minor to 1.6.0. Patch versions are for bugfixes and this is a new feature that's backwards compatible.

Reply:
totally agree! done in [a19fd4f](https://github.com/Skyscanner/cfripper/pull/212/commits/a19fd4fa45034a58f328c598058d522a545a6a5c)